scripts/importLanguages.py

```diff
@@ -64,10 +64,10 @@
 
     completeness = translations/(sources*1.0)
     if completeness < l10n_threshold:
-        print(f'❌\t- {locale} is translated at {round(completeness*100, 2)}%, at least {l10n_threshold*100}% is needed')
+        print(f'KO\t- {locale} is translated at {round(completeness*100, 2)}%, at least {l10n_threshold*100}% is needed')
```

Comment:
These changes are required because unicode chars as ✔ failed on the github action environment.